File: OneCodeTeam/3-Tier-ASPNET-Web-App-wit-Entity-Framework-and-20221/[VB]-3 Tier ASP.NET Web App wit Entity Framework (and Self Tracking Objects)/VB/VBAzureNTierWebRoleWithSession/DataAccessLayer/DataManager.vb
'****************************** Module Header ******************************\
' Module Name:  DataManager.vb
' Project:      DataAccessLayer
' Copyright (c) Microsoft Corporation.
'
' The sample code demonstrates how to build a simple 3-tier Asp.net Web Role, 
' which uses Entity Framework (SQL Azure database) as the Data Access Layer. 
' This sample also shows how to implement a User Session Handling (With Azure Cache Service).
'
' This class is used to retrieve data from Entity Framework (SQL Azure database).
'
' This source is subject to the Microsoft Public License.
' See http://www.microsoft.com/en-us/openness/licenses.aspx#MPL
' All other rights reserved.
'
' THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND, 
' EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED 
' WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
'***************************************************************************/



Public Class DataManager
    Public context As TestDBEntities
    Public Sub New()
        context = New TestDBEntities()
    End Sub

    Public Function GetAllTable() As List(Of TestTable)
        Dim list As New List(Of TestTable)()
        list = context.TestTable.ToList()
        Return list
    End Function
End Class

File: OneCodeTeam/Create-a-three-tier-ASPNET-web-application-with-111749/[C#]-Create a three-tier ASP.NET web application with Entity Framework and Self Track/C#/CSAzureNTierWebRoleWithSession/DataAccessLayer/DataManager.cs
/****************************** Module Header ******************************\
Module Name:  DataManager.cs
Project:      DataAccessLayer
Copyright (c) Microsoft Corporation.
 
The sample code demonstrates how to build a simple 3-tier Asp.net Web Role, 
which uses Entity Framework (SQL Azure database) as the Data Access Layer. 
This sample also shows how to implement a User Session Handling (With Azure Cache Service).

This class is used to retrieve data from Entity Framework (SQL Azure database).

This source is subject to the Microsoft Public License.
See http://www.microsoft.com/en-us/openness/licenses.aspx#MPL
All other rights reserved.
 
THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND, 
EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED 
WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
\***************************************************************************/



using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace DataAccessLayer
{
    public class DataManager
    {
        public TestDBEntities context;
        public DataManager()
        {
            context = new TestDBEntities();
        }

        /// <summary>
        /// Get all data of TestTable table of SQL Azure.
        /// </summary>
        /// <returns></returns>
        public List<TestTable> GetAllTable()
        {
            List<TestTable> list = new List<TestTable>();
            list = context.TestTable.ToList<TestTable>();
            return list;
        }
    }
}
